<?xml version="1.0" encoding="utf-8"?>
<section xmlns="http://docbook.org/ns/docbook" version="5.0" xml:id="zend.exception.using"><info><title>Użycie wyjątków</title></info>

    

    <para>
        Każdy wyjątek wyrzucany przez klasy Zend Framework powinien
        wyrzucać wyjątek, który rozszerza podstawową klasę Zend_Exception.
    </para>

    <example xml:id="zend.exception.using.example"><info><title>Przykład łapania wyjątku</title></info>
        
        <programlisting role="php"><![CDATA[
try {
    Zend_Loader::loadClass('nonexistantclass');
} catch (Zend_Exception $e) {
    echo "Caught exception: " . get_class($e) . "\n";
    echo "Message: " . $e->getMessage() . "\n";
    // inny kod do obsługi błędu.
}

        ]]></programlisting>
    </example>

    <para>
        Zobacz dokumentację dla poszczególnych komponentów Zend Framework
        aby uzyskać bardziej szczegółowe informacje o tym, które metody
        wyrzucają wyjątki, jakie są okoliczności wyrzucenia wyjątku oraz które
        klasy wyjątków pochodzą z klasy Zend_Exception.
    </para>

</section>
